test/notation_test.exs

```diff
@@ -79,6 +84,7 @@ defmodule NotationTest do
       assert Notation.notate(%{"a" => 1}) == "1NaSAH"
     end
 
+    #This fails on master!
```

Comment:
Assume your work hasn't fi it! if it's still failing e should remove test and add something on the README Todo

Reply:
I've commented the tests out, and added better descriptions to the tests, and the readme.